### PR TITLE
section commit for August 11th, '22

### PR DESCRIPTION
**Pull request by croqueGrec09**
finalise database configuration for storage

### CODE REVIEW

File: grails-app/conf/spring/resources.groovy [hunk 1]
```diff
@@ -85,4 +85,8 @@ beans = {
     securityContextPersistenceFilter( SecurityContextPersistenceFilter )
     // .. ]
 
+    //workaround for multi datasource bug, see https://github.com/grails/grails-database-migration/issues/157 (the PR is still open!)
```
Comment (klodav) [inline grails-app/conf/spring/resources.groovy:88]: Code already added .. see above

File: grails-app/views/subscription/stats.gsp [hunk 2]
```diff
@@ -215,7 +215,7 @@
             <ui:tabs class="la-overflowX-auto">
                 <ui:tabsItem controller="subscription" action="stats" params="${params + [tab: 'total']}" text="${message(code: 'default.usage.allUsageGrid.header')}" tab="total"/>
                 <g:each in="${monthsInRing}" var="month">
-                    <ui:tabsItem controller="subscription" action="stats" params="${params + [tab: DateUtils.getSDF_yyyyMM().format(month)]}" text="${DateUtils.getSDF_yyyyMM().format(month)}" tab="${DateUtils.getSDF_yyyyMM().format(month)}"/>
+                    <ui:tabsItem controller="subscription" action="stats" params="${params + [tab: month.format(DateTimeFormatter.ofPattern('yyyy-MM'))]}" text="${month.format(DateTimeFormatter.ofPattern('yyyy-MM'))}" tab="${month.format(DateTimeFormatter.ofPattern('yyyy-MM'))}"/>
```
Comment (klodav) [inline grails-app/views/subscription/stats.gsp:218]: Use DateUtils.getSDF_yyyyMM() - read https://github.com/hbz/laser3/blob/dev/documentation/details.md

Comment (croqueGrec09) [reply grails-app/views/subscription/stats.gsp:218]: month is instance of class java.time.LocalDate whose format() method is not deprecated at all, see https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/LocalDate.html#format(java.time.format.DateTimeFormatter)

File: grails-app/conf/spring/resources.groovy (hunk 1 above)
Comment (croqueGrec09) [reply grails-app/conf/spring/resources.groovy:88]: will be fixed

File: grails-app/migrations/changelog-storage.groovy [hunk 3]
```diff
@@ -2,5 +2,6 @@ databaseChangeLog = {
 
     changeSet(author: "kloberd (generated)", id: "storage") {}
 
+    include file: 'changelogs/2022-08-08.groovy'
```
Comment (klodav) [inline grails-app/migrations/changelog-storage.groovy:5]: rename to 2022-08-08-storage.groovy

File: grails-app/migrations/changelogs/2022-08-08.groovy [hunk 4]
```diff
@@ -0,0 +1,244 @@
+databaseChangeLog = {
```
Comment (klodav) [inline grails-app/migrations/changelogs/2022-08-08.groovy:1]: rename file to 2022-08-08-storage.groovy

**Review by klodav — APPROVED** [review]

**Pull request merged**